app/services/imports/case_logs_import_service.rb

```diff
@@ -142,15 +142,17 @@ def create_log(xml_doc)
       attributes["postcode_known"] = attributes["postcode_full"].nil? ? 0 : 1
 
       # Not specific to our form but required for consistency (present in import)
-      attributes["old_form_id"] = Integer(field_value(xml_doc, "xmlns", "FORM"))
+      attributes["old_form_id"] = safe_string_as_integer(xml_doc, "FORM")
       attributes["created_at"] = Time.zone.parse(field_value(xml_doc, "meta", "created-date"))
       attributes["updated_at"] = Time.zone.parse(field_value(xml_doc, "meta", "modified-date"))
+      attributes["old_id"] = field_value(xml_doc, "meta", "document-id")
 
       # Required for our form invalidated questions (not present in import)
       attributes["previous_la_known"] = 1 # Defaulting to Yes (Required)
       attributes["la_known"] = 1 # Defaulting to Yes (Required)
       attributes["is_la_inferred"] = false # Always keep the given LA
       attributes["first_time_property_let_as_social_housing"] = first_time_let(attributes["rsnvac"])
+      attributes["declaration"] = 1
```

Comment:
Is there not a field for this in the softwire data?

Reply:
fixed :)